src/bindings/python/src/openvino/torch/__init__.py

```diff
@@ -0,0 +1,9 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2018-2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+from openvino.utils import _add_openvino_libs_to_search_path
+
+_add_openvino_libs_to_search_path()
+
+from openvino.frontend.pytorch.torchdynamo import backend
```

Comment:
@jiwaszki, @ilya-lavrenov, we are adding `openvino.torch` as an importable module that auto-registers 'openvino' backend in `torch` and should be imported by the user only if torch is installed. It is the only way to register openvino as a backend until future PR to Pytorch itself is made that registers openvino backend in the stock Pytorch. I would like to have explicit approval from each of you.

Reply:
We are going to merge it into 2023.0 as something that doesn't affect other components. @moslex, FYI.